### DISCUSSION TITLE
[FR] [5.x] Allow entry sidebar to be hidden/collapsed for entry types

### DISCUSSION THREAD

**Discussion by turnstylerj** [craftcms/cms#14445] [Ideas] [answered]
Now that Matrix blocks are entries and edited in the half-screen overlay, things can start getting really cramped with a sidebar that isn't super useful for those types of nested entries. It'd be nice if there was an option when configuring an entry type to collapse/hide the sidebar when it's shown in an overlay.

**Comment by brandonkelly** [comment, marked as answer]
Already made it possible to expand/collapse slideout sidebars, for the next release :) (#14418)
